conan/tools/microsoft/msbuilddeps.py

```diff
@@ -42,6 +44,7 @@ class MSBuildDeps(object):
           <PropertyGroup>
             <LocalDebuggerEnvironment>PATH=%PATH%;$(Conan{name}BinaryDirectories)$(LocalDebuggerEnvironment)</LocalDebuggerEnvironment>
             <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
+            <CAExcludePath>$(Conan{name}CAExcludeDirectories);$(CAExcludePath)</CAExcludePath>
```

Comment:
Put the conditional here, to define it or not (by default, this line shouldn't appear)

Reply:
why do I need condition here? `$(Conan{name}CAExcludeDirectories)` will just resolve to an empty string, so line will have no effect.